core/controlplane/config/config.go

```diff
@@ -255,11 +257,6 @@ func NewDefaultCluster() *Cluster {
 		EtcdSettings: EtcdSettings{
 			Etcd: model.NewDefaultEtcd(),
 		},
-		FlannelSettings: FlannelSettings{
```

Comment:
I move PodCIDR and ServiceCIDR into KubeClusterSettings so that they can be accessed in the cloud-config on the nodes (so we can make them roll).  I happen to think that they are KubeClusterSettings anyway and there was little to be gained by having the FlannelSettings type at all.